spec/glossary.md

```diff
@@ -0,0 +1,2 @@
+ - **Storage Provider** - A Service Provider that ofers a cloud-based data storage service conformant with the LWS protocol.
+ - **Identity Provider** - The party that manages the subscriber’s primary authentication credentials and issues assertions derived from those credentials ; see [NIST definition](https://csrc.nist.gov/glossary/term/identity_provider)
```

Comment:
Using `<dfn>` instead of a generic `<strong>` will have many good properties, such as being able to link to the definition easily (from withing this spec, or from other specs). At some point, we will also probably want to link to definitions from the LWS protocol instead.

For example, any use of `<a>Storage Provider</a>` will automatically add a `href` to the definition.

```suggestion
 - <dfn>Storage Provider</dfn> - A Service Provider that ofers a cloud-based data storage service conformant with the LWS protocol.
 - <dfn>Identity Provider</dfn> - The party that manages the subscriber’s primary authentication credentials and issues assertions derived from those credentials ; see [NIST definition](https://csrc.nist.gov/glossary/term/identity_provider)
```